src/amp-story-player/amp-story-player-impl.js

```diff
@@ -434,6 +436,21 @@ export class AmpStoryPlayer {
     });
   }
 
+  /**
+   * Gets publisher configuration for a custom system UI.
+   * @private
+   * @return {?JsonObject}
+   */
+  getCustomUI_() {
```

Comment:
We now know this will evolve into a broader Player configuration, can we update this method name?